test/functional/feature_rbf.py

```diff
@@ -624,6 +627,24 @@ def test_no_inherited_signaling(self):
         assert_equal(True, self.nodes[0].getmempoolentry(optin_parent_tx['txid'])['bip125-replaceable'])
         assert_raises_rpc_error(-26, 'txn-mempool-conflict', self.nodes[0].sendrawtransaction, replacement_child_tx["hex"], 0)
 
+    def test_replacement_relay_fee(self):
+        outpoint = make_utxo(self.nodes[0], int(1.1*COIN))
```

Comment:
any reason to not use miniwallet, like in the previous test?

Reply:
Well I think it makes the test a bit longer because it doesn't give you an outpoint but an utxo ? Though I took it anyway.

Reply:
heh, the point is to create the tx with the miniwallet. In that case the test should be three lines or so.

Reply:
Okay I did drop the miniwallet usage as AFAICT its `fee_rate` argument doesn't allow the feerate granularity (sat per KvB) required to hit the yet-uncovered branch ? Or at least without modifying node's default `incrementalrelayfee`